salt/proxy/netmiko_px.py

```diff
@@ -187,13 +187,19 @@
 import time
 
 import salt.utils.args
+from salt.utils.versions import LooseVersion as _LooseVersion
 
 try:
     from netmiko import ConnectHandler
-    from netmiko.ssh_exception import (
-        NetMikoAuthenticationException,
-        NetMikoTimeoutException,
-    )
+    from netmiko import __version__ as found_netmiko_version
+
+    if _LooseVersion(found_netmiko_version) <= _LooseVersion("3.4.0"):
```

Comment:
I'd do a simple `try/catch ImportError`, it's faster and doesn't require version comparisons.
Try to import from the newer module, fallback to the old.